react-search-component/my-app/src/SearchableList Components/TextDisplay.tsx

```diff
@@ -0,0 +1,26 @@
+type TextDisplayProps = {
+  list: { id: string; sentence: string }[];
+  searchText: string;
+};
+
+export default function TextDisplay({ list, searchText }: TextDisplayProps) {
+  const filteredList = list.filter(({ sentence }) => {
+    return sentence.toLowerCase().includes(searchText.toLowerCase());
```

Comment:
You don't seem to like the shorthand syntax for returning values from an arrow function:
```
const filteredList = list.filter(({ sentence }) =>
    sentence.toLowerCase().includes(searchText.toLowerCase()));
```

Reply:
just a habit to use this way ill try to use the shorthand later on